src/client/components/AddNewGroupComponent/AddNewGroup.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './AddNewGroupStyle.css';
+
+const AddNewGroup = (props) => {
+  let groupname = '';
+
+  if (props.addGroup) {
+    groupname = (
+      <div className="g-name">
+        <h3>Group name</h3>
+        <div>{props.addGroup}</div>
+      </div>
+    );
+  }
+  return (
+    <div className="container">
+      <div className="wraper">
+        <div className="add-group">
+          <button type="button" onClick={props.groupdHandler}>
+            AddGroup
+          </button>
+          <input
+            type="text"
+            value={props.newGroup}
+            placeholder="Add new group"
+            onChange={props.AddGroupName}
+          />
+          {groupname}
+        </div>
+      </div>
+    </div>
+  );
+};
+AddNewGroup.propTypes = {
+  addGroup: PropTypes.string.isRequired,
+  groupdHandler: PropTypes.func.isRequired,
```

Comment:
All functions that are passed should start with `on`. E.g. `onAdd` should be used.